.github/workflows/linux-eic-shell.yml

```diff
@@ -544,14 +544,22 @@ jobs:
         path: ref/
         name: rec_${{ matrix.particle }}_1GeV_20GeV_${{ matrix.detector_config }}.edm4eic.root
         if_no_artifact_found: warn
+    - name: Check if epic-capybara available
+      id: check_capybara_available
+      run: |
+        if which capybara ; then
+          echo "found=false" >> ${GITHUB_OUTPUT}
+        else
+          echo "found=true" >> ${GITHUB_OUTPUT}
+        fi
     - name: Checkout epic-capybara
       uses: actions/checkout@v3
-      if: steps.download_previous_artifact.outputs.found_artifact == 'true'
+      if: steps.download_previous_artifact.outputs.found_artifact == 'true' && steps.check_capybara_available.outputs.found == 'false'
       with:
         repository: 'eic/epic-capybara'
         path: epic-capybara
     - name: Build and install epic-capybara
-      if: steps.download_previous_artifact.outputs.found_artifact == 'true'
+      if: steps.download_previous_artifact.outputs.found_artifact == 'true' && steps.check_capybara_available.outputs.found == 'false'
```

Comment:
Would it make sense to remove this entirely? In that case, will eic-container deploy nightly capybara automatically for us?

Reply:
If we want to run the 'old' bara.py, then we need this. We don't install the 'old' versions in the container.

Reply:
The bara.py comes from the checkout above, there is no plan to install it with wheels.

Reply:
`capybara` is now installed inside the container (by git hash, currently [3a6e6a9c75630fd5e7375a8bd7f9b5c20eb227cd](https://github.com/eic/epic-capybara/commit/3a6e6a9c75630fd5e7375a8bd7f9b5c20eb227cd)), so I'm going to remove this check (make it a requirement instead) and avoid the hatching of the a checked out version.